### PR TITLE
The image cover was added in the detail

### PR DESCRIPTION
**Pull request by pabloqv10**
## Summary

- The image cover was added in the detail

## Trello Card

[https://trello.com/c/b6KE5ia4/27-detalle-de-libro-ii-imagen](url)

## Screenshot
detail-2-image

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -38,3 +46,9 @@ $gray : #828282;
   color: $gray;
   padding: 0 8px;
 }
+
+.book-cover {
+  height: 368px;
+  width: 261px;
+  flex-direction: row;
+}
```
Comment (manueltuero) [inline scss/detail.scss]: Add a new empty line here.

File: index.html [hunk 2]
```diff
@@ -10,24 +10,29 @@
...
-    </div>
-    <div class="row">
-        <h3 class="year">Año de publicación:</h3>
-        <p class="description">Año de publicación</p>
+    <div class="info">
+      <div class="row">
+          <h1 class="title">Título del libro</h1>
+          <p class="gender">(género)</p>
+      </div>
+      <div class="row">
```
Comment (manueltuero) [inline index.html]: Maybe you can avoid using a div like a separator. Using **margin** for example.

File: index.html [hunk 3]
```diff
@@ -10,24 +10,29 @@
 </head>
 <body>
   <div class="book">
-    <div class="row">
-        <h1 class="title">Título del libro</h1>
-        <p class="gender">(género)</p>
+    <div class="image">
+      <img src="./assets/book-cover.png" alt="book-cover" class="book-cover">
```
Comment (manueltuero) [inline index.html]: You can write a normal text in the **alt** attribute. E.g.: **"Book cover"**

File: index.html [hunk 4]
```diff
@@ -10,24 +10,29 @@
...
+    <div class="info">
+      <div class="row">
+          <h1 class="title">Título del libro</h1>
+          <p class="gender">(género)</p>
+      </div>
+      <div class="row">
+        <!-- TODO: separador -->
+      </div>
+      <div class="row">
+          <h3 class="author">Autor del libro:</h3>
```
Comment (manueltuero) [inline index.html]: I don't think that must use an h3 tag here

**Review by manueltuero — CHANGES_REQUESTED** [review]

**Review by Dorian30 — APPROVED** [review]

**Review by manueltuero — APPROVED** [review]

**Pull request merged**